webshare/idlharness.https.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare IDL test</title>
+    <link rel="help" href="https://wicg.github.io/web-share/">
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/resources/WebIDLParser.js></script>
+    <script src=/resources/idlharness.js></script>
+  </head>
+  <body>
+    <script type="text/plain" id="untested">
+        interface Navigator {
+        };
+    </script>
+    <script type="text/plain" id="tested">
+        partial interface Navigator {
+          [SecureContext]
+          Promise<void> share(ShareData data);
+        };
+
+        dictionary ShareData {
+          USVString title;
+          USVString text;
+          USVString url;
+        };
+    </script>
+    <script>
+        "use strict";
+        var idl_array = new IdlArray();
+        idl_array.add_untested_idls(document.querySelector("#untested").textContent);
```

Comment:
Doesn't matter, but it's fine to do `idl_array.add_untested_idls("interface Navigator {};");` if you like after the big IDL block is gone from the file.